### PR TITLE
Fix a typo: user menu registry example

### PR DESCRIPTION
**Pull request by beshoynabeih**
fix errors in javascript code example.

### CODE REVIEW

File: content/developer/reference/frontend/registries.rst [hunk 1]
```diff
@@ -298,7 +298,7 @@ Example:
     registry.category("user_menuitems").add("my item", (env) => {
         return {
             description: env._t("Technical Settings"),
-            callback: () => { env.services.action_manager.doAction(3); };
+            callback: () => { env.services.action_manager.doAction(3); },
             hide: (Math.random() < 0.5),
-        };
-    }
+        }
+    })
```
Comment (brboi) [inline content/developer/reference/frontend/registries.rst:304]:
```suggestion
        };
    });
```

**Review by brboi — CHANGES_REQUESTED** [review]
It's nice you spotted this typo.
Thank you for helping us improve the user documentation of Odoo!

Can you please make the requested change?

File: content/developer/reference/frontend/registries.rst (hunk 1 above)
Comment (beshoynabeih) [reply content/developer/reference/frontend/registries.rst]: sure, I did it.

**Comment by beshoynabeih** [comment]
@brboi should I do anything else to merge my request?

**Comment by brboi** [comment]
> @brboi should I do anything else to merge my request?

There is one last step required: a validation from @odoo/doc-review

😉

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Hello @beshoynabeih, could you please squash your commits into one so we can proceed with the merge? :)
Ideally, the commit title would be something like `[FIX] developer/frontend: typo in the user menu registry example`.

**Comment by AntoineVDV** [comment]
> @AntoineVDV I did the squash could you please check you

@beshoynabeih As you can see [here](https://github.com/odoo/documentation/pull/1975/commits) your pull request counts 5 commits. After squashing the commits into one, it should only have a single commit.

**Comment by beshoynabeih** [comment]
@AntoineVDV thank you for helping. now it is one commit

**Review by AntoineVDV — APPROVED** [review]
Good, thank you!

@robodoo r+

**Pull request closed without merging**